helper_test.go

```diff
@@ -525,7 +525,7 @@ func createPolicySetParameter(t *testing.T, client *Client, ps *PolicySet) (*Pol
 	}
 }
 
-func createPolicySet(t *testing.T, client *Client, org *Organization, policies []*Policy, workspaces []*Workspace) (*PolicySet, func()) {
+func createPolicySet(t *testing.T, client *Client, org *Organization, policies []*Policy, workspaces []*Workspace, kind PolicyKind) (*PolicySet, func()) {
```

Comment:
I wonder if it's worth creating a new func here instead of modifying the signature of this one. e.g. 

```
func createPolicySetWithKin(t *testing.T, client *Client, org *Organization, policies []*Policy, workspaces []*Workspace, kind PolicyKind) (*PolicySet, func()) {
...
```

I'm thinking that this would then remove the need modify all of the existing tests (by adding `,""`  at the end of the func calls) and make a clear distinction between beta and non-beta test setup.

... but ... the end result is probably the same so I'm not 100% sure that it's worth it right now.  What do you think?

Reply:
My initial thoughts were that the CreatePolicySet test helper should probably be generic to create a Policy set of either type and the caller should have the option to create any `kind` of policy set they want to create.

Also when OPA becomes GA we would probably have to modify the existing tests or have two funcs (`createPolicySet` and `createPolicySetWithKind`) which does the same thing?

I am ok to go either way tbh :) `createPolicySetWithKind` would make this PR cleaner , but leaving as is would mean less work for the future me hehe

Reply:
Yeah ... Let's leave it as is for now